### PR TITLE
Jason modules hangman

### PR DESCRIPTION
**Pull request by jasonkx9**
Added the Hangman Module

### CODE REVIEW

File: src/data/ModuleData.js [hunk 1]
```diff
@@ -129,6 +129,7 @@ markdown: `
     title: "JS Comments",
     markdown: `
 # Learning Module 4
+Hangman Module
```
Comment (CSegotaDev) [inline src/data/ModuleData.js]: is this supposed to be here? looks like this change was made in the middle of a different data object?

Comment (jasonkx9) [reply src/data/ModuleData.js]: No, you're right that shouldn't be there. I'm guessing I was planning to replace an earlier module and realized it was a much better idea just to add one to the end and forgot to delete that.

File: src/data/ModuleData.js [hunk 2]
```diff
@@ -361,6 +362,16 @@ when you need the value of a variable to change frequently
     embeddedIDEURL:
       "https://replit.com/@jdaus/HurtfulSpectacularGnuassembler?lite=true",
     customIFrameStyle: null
+  },
+  {
+    id: 15,
+    title: "Hangman Module",
+    markdown: `
+    # Hangman Module
```
Comment (CSegotaDev) [inline src/data/ModuleData.js]: I feel like something else could be said in the guide section about the module. But at a minimum might want to say something like "Continue on to the Engagement Tab to play Programmer Hangman!" ( or something )

Other ideas for things that you could add here:
* In this module, you'll have x lives to try and guess a programming related keyword.
* There are x topics
* If you get stuck you can ask for a hint!
* (get creative!)
* (or don't, not a big deal either way)

File: src/data/ModuleData.js (hunk 1 above)
Comment (jasonkx9) [reply src/data/ModuleData.js]: Just fixed that.

**Pull request closed without merging**